lib/base-controller.js

```diff
@@ -65,7 +65,11 @@ module.exports = class BaseController extends Controller {
   locals(req, res) {
     const locals = super.locals(req, res);
     const stepLocals = this.options.locals || {};
-    const fields = _.map(this.options.fields, (field, key) => ({key, mixin: field.mixin}));
+    const fields = _.map(this.options.fields, (field, key) => ({
+      key,
```

Comment:
what's happening here? is `key` an object, and does the map iterator return anything? I'm finding it hard to read.